src/utils/staticTokenDefinition.ts

```diff
@@ -0,0 +1,99 @@
+import {
+  Address,
+  BigInt,
+} from "@graphprotocol/graph-ts"
+  
+// Initialize a Token Definition with the attributes
+export class StaticTokenDefinition {
+  address : Address
+  symbol: string
+  name: string
+  decimals: BigInt
+
+  // Initialize a Token Definition with its attributes
+  constructor(address: Address, symbol: string, name: string, decimals: BigInt) {
+    this.address = address
+    this.symbol = symbol
+    this.name = name
+    this.decimals = decimals
+  }
+
+  // Get all tokens with a static defintion
+  static getStaticDefinitions(): Array<StaticTokenDefinition> {
+    let staticDefinitions = new Array<StaticTokenDefinition>(6)
+
+    // Add DGD
+    let tokenDGD = new StaticTokenDefinition(
+      Address.fromString('0xe0b7927c4af23765cb51314a0e0521a9645f0e2a'),
```

Comment:
Nothing against having these tokens here, just curiosity.
Any idea why these tokens are here?

If you don't know that's fine.

Reply:
im also curious

Reply:
I don't know. I would suggest to remove them. But first I want to perform some tests on real contracts to ensure we can fetch that information doing a proper contract call.